decidim-core/app/validators/translated_etiquette_validator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# A custom validator to check for presence in I18n-enabled fields. In order to
```

Comment:
This comment seems a copypaste from TranslatablePresenceValidator, can you adapt it to TranslatedEtiquetteValidator 🙏🏽 ?

Reply:
fixed in 4596c843b7